openhands/runtime/client/client.py

```diff
@@ -649,52 +647,12 @@ async def list_files(request: Request):
             if not os.path.exists(full_path) or not os.path.isdir(full_path):
                 return []
 
-            # Check if .gitignore exists
-            gitignore_path = os.path.join(full_path, '.gitignore')
```

Comment:
I still think, at least the gitignore check should not be removed for the UI here.

Reply:
The client is running inside the container and still has access to the mounted workspace folder, no?
Or are you thinking of the `remote` runtime? 🤔

Reply:
Thanks, that indeed makes it clearer. :)